README.md

```diff
@@ -115,7 +115,7 @@ stripe-mock
 Run all tests:
 
 ```sh
-dotnet test
+dotnet test ./src/StripeTests/StripeTests.csproj
```

Comment:
Hm, I guess that's better since it's fair to assume the command would be run from the repo's root, but we should change the other two commands below then.

Also, there's no need to specify the `.csproj` file, just the directory is enough:
```
dotnet test ./src/StripeTests
```

I'm not sure forward slashes would work on Windows though 🤔 Maybe let's keep the commands as is and just add a note saying they should be run from the `src/StripeTests` directory?

Reply:
Sounds good :)